airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -210,6 +210,8 @@ public AirbyteCatalog discover(final JsonNode config) throws Exception {
       catalog.setStreams(streams);
     }
 
+    // We can *potentially* upgrade here. However, this will be an issue since we will likely incur data loss
```

Comment:
teeechnically it's exactly the same data loss as what's happening today :shrug: so it wouldn't cause any regressions

Reply:
Yeah - this is an option. But inclined to do it the correct way.